flowclient/flowclient/client.py

```diff
@@ -455,6 +455,320 @@ def daily_location(
     }
 
 
+def _meaningful_locations(
+    start_date: str,
+    stop_date: str,
+    label: str,
+    labels: Dict[str, Dict[str, dict]],
```

Comment:
What do `label` and `labels` refer to? It's somewhat confusing that these variables have such similar names, can we make them more explicit (or provide a docstring to explain them)?

Reply:
(At first I wasn't even sure if these are alternative ways of providing a value for the same thing.)